src/content/protocols/aave.md

```diff
@@ -0,0 +1,998 @@
+| stkAAVE (Implementation)                                          | redeemOnBehalf                         | Allows the `CLAIM_HELPER_ROLE` to redeem the staked Aave (underlying) for the stkAAVE tokens for any user that is in the cooldown window. No checks are in place to prevent the Governor from withdrawing user funds to a 3rd party or seize the assets.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | `CLAIM_HELPER_ROLE`                                                                                                                                                          |
+| stkAAVE (Implementation)                                          | claimRewardsOnBehalf                   | Allows the `CLAIM_HELPER_ROLE` to claim emissioned rewards on behalf of a user. No checks are in place to prevent the Governor from claiming rewards of a user to a 3rd party.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
```

Comment:
make section "Risk parameter control"

make column "Guardrails enforced by" and switch with "Controlled by" column as in the table in the Governance section